roadmap/implementers-guide/src/types/overseer-protocol.md

```diff
@@ -60,13 +60,16 @@ Messages to and from the availability store.
 ```rust
 enum AvailabilityStoreMessage {
 	/// Query the PoV of a candidate by hash.
```

Comment:
I find it a bit unclear what `AvailableData` here means? The description still refers to the `PoV`, a bit of elaboration would be helpful

Reply:
that's defined in `availability.md`